### DISCUSSION TITLE
[Esame 23 Gennaio 2019] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#223] [Esami - Bertini @ Informatica]
![immagine](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/0116549f-91ef-4591-a2f2-e5af302b8795)

- Simile a #165

**Comment by LeoCosta02** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/128373356/1f0a4321-cb19-4876-9d76-0839f6bbac5a)

**Comment by LeoCosta02** [comment]
Dovrebbe essere giusto essendo simile ai primi due punti dell'esercizio2 foglio 5

**Comment by Elia-Belli** [comment]
Probabilità di pescare le varie tipolegie di moneta: $\mathbb{P}(TT)=5/10,\ \mathbb{P}(CC)=3/10,\ \mathbb{P}(TC)=2/10$

**(1)** Posso ottenere testa solamente se pesco una moneta con entrambe teste o una normale, quindi la partizione dell'evento di pescare una moneta e ottenere testa si riduce a:

$$\mathbb{P}(T)=\mathbb{P}(TT)\cdot 1 + \mathbb{P}(TC)\cdot \frac{1}{2}=\frac{5}{10}\cdot 1+\frac{2}{10}\cdot \frac{1}{2}=\frac{3}{5}$$

**(2)** Applicando Bayes:

$$\mathbb{P}(TT|T)=\frac{\mathbb{P}(TT\cap T)}{\mathbb{P}(T)}=\frac{\frac{5}{10}\cdot 1}{\frac{3}{5}}=\frac{5}{6}$$

**Reply by neoSnakex34** [reply]
è corretto ma non hai applicato bayes nel 2) a quanto vedo, hai solo seguito la formula di probabilità condizionata classica
